pkgs/development/python-modules/crc/default.nix

```
@@ -0,0 +1,17 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "crc";
+  version = "4.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-W1ElEk1127TSXsEHvaU/32zapdY8dH82CWdBNY9r2j4=";
+  };
+
+  meta = with lib; {
+    description = "Python module for calculating and verifying predefined & custom CRC's";
```

```suggestion
    changelog = "https://github.com/Nicoretti/crc/releases/tag/${version}";
    description = "Python module for calculating and verifying predefined & custom CRC's";
```

Please add a changelog pattern, helps with generating links in automated updates.